libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java

```diff
@@ -401,9 +404,10 @@ private void initializeNavigation(Context context, NavigationOptions options, Lo
 
   private void addNavigationListeners() {
     navigation.registerRouteProgressObserver(navigationViewVm);
-    navigation.unregisterLocationObserver(navigationViewVm);
+    navigation.registerLocationObserver(navigationViewVm);
```

Comment:
Can we remove this line? We have it in other PR. This will cause merge conflicts.

Reply:
Same as above.